test/python/test_axpy_pipelined.py

```diff
@@ -13,7 +13,7 @@
 
 # For eg. when running the test from ngraph-bridge/build_cmake/test/python
 # you can add this path as below
-#sys.path.insert(0, '../../examples')
+sys.path.insert(0, '../../examples')
```

Comment:
This is uncommented to test locally, so we can comment it back

Reply:
there might be no harm having it there and helps when running locally. What do you think? Or could it interfere when running elsewhere?